src/features/UserAuth/services/validationSchemas.js

```diff
@@ -1,71 +1,71 @@
-import * as Yup from 'yup'
+import * as Yup from 'yup';
 import {
-    minNameLength,
-    maxNameLength,
-    minPassLength,
-} from '../../../services/constants'
+  minNameLength,
+  maxNameLength,
+  minPassLength,
+} from '../../../services/constants';
 
 function equalTo(ref, msg) {
-    return Yup.mixed().test({
-        name: 'equalTo',
-        exclusive: false,
-        // eslint-disable-next-line no-template-curly-in-string
-        message: msg || '${path} must be the same as ${reference}',
-        params: {
-            reference: ref.path,
-        },
-        test(value) {
-            return value === this.resolve(ref)
-        },
-    })
+  return Yup.mixed().test({
+    name: 'equalTo',
+    exclusive: false,
+    // eslint-disable-next-line no-template-curly-in-string
+    message: msg || '${path} must be the same as ${reference}',
```

Comment:
если заменишь кавычки - вроде можно будет убрать это правило
```suggestion
    message: msg || `${path} must be the same as ${reference}`,
```

Reply:
не, тогда это становится строкой с подстановкой где path с reference переменные, которых нет в области видимости